sw/device/silicon_creator/manuf/base/ft_personalize.c

```diff
@@ -695,6 +697,10 @@ static status_t personalize_endorse_certificates(ujson_t *uj) {
   // a perso LTV object. Reset the `next_cert` pointer and `free_room` size.
   next_cert = all_certs;
   free_room = sizeof(all_certs);
+  // 1K should be enough for the largest certificate perso LTV object.
+  const size_t kBufferSize = 1024;
+  alignas(uint32_t) uint8_t buffer[kBufferSize];
```

Comment:
Should this be a static scratch buffer instead of being placed on the stack? It is pretty large